graphql/patch_resolver.go

```diff
@@ -103,6 +105,49 @@ func (r *patchResolver) Duration(ctx context.Context, obj *restModel.APIPatch) (
 	return makePatchDuration(timeTaken, makespan), nil
 }
 
+// GeneratedTaskCounts is the resolver for the generatedTaskCounts field.
+func (r *patchResolver) GeneratedTaskCounts(ctx context.Context, obj *restModel.APIPatch) (map[string]interface{}, error) {
+	patchID := utility.FromStringPtr(obj.Id)
+	p, err := patch.FindOneId(patchID)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while finding patch with id: `%s`: %s", patchID, err.Error()))
+	}
+	if p == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Patch does not exist: `%s`", patchID))
+	}
+	proj, _, err := model.FindAndTranslateProjectForPatch(ctx, evergreen.GetEnvironment().Settings(), p)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding project config for patch '%s': %s", patchID, err.Error()))
```

Comment:
nit: should we use a sentence with capitalization like in other parts of the function